src/middleware/checkLoggedIn.js

```diff
@@ -11,19 +11,9 @@ module.exports = (req, res, next) => {
   if (access) {
     verifyToken(req.cookies)
       .then(() => next())
-      .catch(err =>
```

Comment:
Wasn't it like this originally? Why did it get changed - if there was a reason, will changing it back maybe break something else?

Reply:
as in, we had it doing a redirect which then got changed into a `res.send` at some point

Reply:
i'm not actually sure why it got changed i don't really remember, but i do know that we were doing `res.send` just cause we had the form's validation or Axios which is doing the request to the server and not the the browser